tests/app/organisation/test_rest.py

```diff
@@ -1290,3 +1290,133 @@ def test_notify_org_users_of_request_to_go_live_requires_org_user_permission(
 
     assert mock_persist_notification.call_args_list == []
     assert mock_send_notification_to_queue.call_args_list == []
+
+
+def test_notify_org_member_about_continuation_of_go_live_request(
+    mocker,
+    admin_request,
+    notify_service,
+    sample_organisation,
+    sample_service,
+    organisation_continue_go_live_request_template,
+    hostnames,
+):
+    notify_service = dao_fetch_service_by_id(current_app.config["NOTIFY_SERVICE_ID"])
+
+    go_live_user = create_user(email="go-live-user@example.gov.uk", name="Go live user")
+    first_org_user = create_user(email="first-org-user@example.gov.uk", name="First org user")
+    second_org_user = create_user(email="second-org-user@example.gov.uk", name="Second org user")
+    dao_add_user_to_organisation(
+        organisation_id=sample_organisation.id, user_id=first_org_user.id, permissions=["can_make_services_live"]
+    )
+    dao_add_user_to_organisation(
+        organisation_id=sample_organisation.id, user_id=second_org_user.id, permissions=["can_make_services_live"]
+    )
+
+    notifications = [object()]
+
+    mock_persist_notification = mocker.patch(
+        "app.organisation.rest.persist_notification",
+        side_effect=notifications,
+    )
+    mock_send_notification_to_queue = mocker.patch(
+        "app.organisation.rest.send_notification_to_queue",
+    )
+    sample_service.organisation = sample_organisation
+    sample_service.has_active_go_live_request = True
+    sample_service.go_live_user = go_live_user
+
+    admin_request.post(
+        "organisation.notify_org_member_about_continuation_of_go_live_request",
+        service_id=sample_service.id,
+        _data={"to": "first-org-user@example.gov.uk", "service_name": "test service", "body": "blah blah blah"},
+        _expected_status=204,
+    )
+
+    assert mock_persist_notification.call_count == 1
+    assert mock_persist_notification.call_args_list[0][1]["recipient"] == "first-org-user@example.gov.uk"
+
+    assert mock_persist_notification.call_args_list[0][1] == dict(
+        template_id=uuid.UUID("62f12a62-742b-4458-9336-741521b131c7"),
```

Comment:
[question] we prefer to hardcode the template id here rather than access it from config? (same for the other test)

Reply:
yep, I'm generally fairly OK hard-coding values in tests, especially in these cases where they should never change anyway and if they do change I'd quite like the test to break to require extra validation of intent.